tests/nested/manual/fde-on-classic/gadget.yaml

```diff
@@ -0,0 +1,55 @@
+volumes:
+  pc:
+    # bootloader configuration is shipped and managed by snapd
+    bootloader: grub
+    structure:
+      - name: mbr
+        type: mbr
+        size: 440
+        update:
+          edition: 1
+        content:
+          - image: pc-boot.img
+      - name: BIOS Boot
+        type: DA,21686148-6449-6E6F-744E-656564454649
+        size: 1M
+        offset: 1M
+        offset-write: mbr+92
+        update:
+          edition: 2
+        content:
+          - image: pc-core.img
+      - name: EFI System partition
+        filesystem: vfat
+        # UEFI will boot the ESP partition by default first
+        type: EF,C12A7328-F81F-11D2-BA4B-00A0C93EC93B
+        size: 99M
```

Comment:
(nitpick) I guess we want something here that is close to what the distro picks for the ESP size (which is probably/maybe smaller) but it's fine for now

Reply:
Right, but we also need to change sfdisk params in mk-image.sh if we change this.

Reply:
what should we set it to?

Reply:
Maybe better to leave as a follow-up, otherwise you need to change all offsets for sfdisk and it will be an annoyance.

Reply:
yea, I can leave a TODO